adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
I believe OSX supports the same three keep-alive options, it just has a different name for one of them (`IPPROTO_TCP, TCP_KEEPALIVE` instead of `SOL_TCP, TCP_KEEPIDLE`. Of course, I still need to test this...